apps/communications/contacts/test/unit/contacts_test.js

```diff
@@ -301,6 +301,24 @@ suite('Contacts', function() {
 
         window.ActivityHandler.currentlyHandling = false;
       });
+
```

Comment:
we woudl need another test that test that with other activityName or other activityType no class is added